ui/app/templates/components/list-item.hbs

```diff
@@ -3,33 +3,27 @@
 {{else if linkParams}}
   <LinkedBlock @params={{linkParams}} @class="list-item-row">
     <div class="level is-mobile">
-      <div class="level-left">
-        <div>
-          {{#link-to params=linkParams class="has-text-weight-semibold"}}
-            {{yield (hash content=(component "list-item/content"))}}
-          {{/link-to}}
-        </div>
-      </div>
+      <div class="level-left is-flex-1">
+        {{#link-to params=linkParams class="has-text-weight-semibold has-text-black is-display-flex is-flex-1 is-no-underline"}}
```

Comment:
whew, that's quite the class list. 😉

Reply:
😬 Yeah - hoping to move away from this - or at least hide it under the rug with more components